.github/workflows/build.yaml

```diff
@@ -32,7 +32,9 @@ jobs:
           restore-keys: |
             mkdocs-material-
       - run: pip install -r requirements.txt
-      - run: mkdocs gh-deploy -s --force
+      - run: |
+          mike set-default 0.6.1
+          mike deploy --push 0.7.0 -t "0.7.0 (dev)"
```

Comment:
thinking: might switch this alias to just `dev` at some point. But I do sort of like having a ref to what the next version being baked might be 🤷